### DISCUSSION TITLE
https://klp1.com/issues/wrong-time/

### DISCUSSION THREAD

**Discussion by giscus** [blakadder/klp1#7] [Announcements]
# https://klp1.com/issues/wrong-time/

Fix wrong time and/or date that's showing in history, logs and console

https://klp1.com/issues/wrong-time/

**Comment by zemuro** [comment]
And yes it worked! Super.

**Comment by waintdeir** [comment]
I leave this here, because I was searching a bit for this:
You can set your own local NTP server as prefered server under /etc/chrony/chrony.conf by adding the line
> server 0.0.0.0 iburst prefer

If you want it to use only your server just add a **"#"** in front of the line starting with **"pool"**:
> #pool 2.debian.pool.ntp.org iburst

To edit the file type
> nano /etc/chrony/chrony.conf

To save the changes press **CTRL+X** and confirm by pressing **Y**

**Comment by orhanatci** [comment]
I need videao about that please...
